docs/standard/microservices-architecture/multi-container-microservice-net-applications/microservice-application-design.md

```diff
@@ -57,43 +57,36 @@ Each microservice has its own database, allowing it to be fully decoupled from o
 
 ### eShopOnContainers: A reference application for .NET Core and microservices deployed using containers
 
-So that you can focus on the architecture and technologies instead of thinking about a hypothetic business domain that you might not know, we have selected a well-known business domain—namely, a simplified e-commerce (e-shop) application that presents a catalog of products, takes orders from customers, verifies inventory, and performs other business functions. This container-based application source code is available in the [eShopOnContainers](https://aka.ms/MicroservicesArchitecture) GitHub repo.
+So that you can focus on the architecture and technologies instead of thinking about a hypothetic business domain that you might not know, we have selected a well-known business domain�namely, a simplified e-commerce (e-shop) application that presents a catalog of products, takes orders from customers, verifies inventory, and performs other business functions. This container-based application source code is available in the [eShopOnContainers](http://aka.ms/MicroservicesArchitecture) GitHub repo.
```

Comment:
Links should use "https" where supported.

Reply:
Yep. mostly on additional resources.

Included http in my checklist!

Fixed the Unicode error too.